.gitignore

```diff
@@ -11,4 +11,4 @@ src/main/resources/docs/
 # MacOS custom attributes files created by Finder
 .DS_Store
 *.iml
-bin/
```

Comment:
Personally, I think it would be better to keep bin/ ignored from git. as it is MacOS custom attributes files created by Finder. It probably would be better to exclude these files and focus on source code for the project so that your repository is "cleaner" and easier to be managed. Otherwise, the number of changed files is huge and harder for mainenance.